### DISCUSSION TITLE
Updates to the rucsoundings web page coming soon

### DISCUSSION THREAD

**Discussion by Brian-Jamison** [NOAA-GSL/GSL-Regional-Model-Forum#37] [Announcements]
The source for RAOB data displayed on our soundings page at https://rucsoundings.noaa.gov/ will soon be changed to PREPBUFR data, provided by NCEP.  The PREPBUFR data are more comprehensive, including more stations and levels. However, please be aware that some levels will have missing height data, possibly including the mandatory pressure levels.  Also, a larger time lag will be noticed with this change, which is due to the processing and delivering of the data. Soundings will generally not be available until 7 hours after their observation time. GSL previously ingested raw data through NOAAPORT, and processed the data in-house. Because of the ingest and processing of NOAAPORT RAOBs is no longer supported in GSL, the decision was made to switch to the PREPBUFR RAOBs.

**Comment by FlyCrestline** [comment]
Thanks Brian,
Is the BAK40 model dead? It used to be spot on for my use.

**Comment by Brian-Jamison** [comment]
Beginning today, the new rucsoundings web page has been put into place.  Some options are no longer available, such as the Bak40 data source, which used the old RUC model that is no longer running.  Also the options for Simple Java Plot, Java Plot, and Unsigned Plot have been removed.

Please let me know of any issues or concerns.
